fabric-convention-tags-v1/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java

```diff
@@ -0,0 +1,257 @@
+		translationBuilder.add(ConventionalBlockTags.LIGHT_GRAY_DYED_BLOCKS, "Light Gray Dyed Blocks");
+		translationBuilder.add(ConventionalBlockTags.LIME_DYED_BLOCKS, "Lime Dyed Blocks");
+		translationBuilder.add(ConventionalBlockTags.MAGENTA_DYED_BLOCKS, "Magenta Dyed Blocks");
+		translationBuilder.add(ConventionalBlockTags.ORANGE_DYED_BLOCKS, "Orange Dyed Blocks");
+		translationBuilder.add(ConventionalBlockTags.PINK_DYED_BLOCKS, "Pink Dyed Blocks");
+		translationBuilder.add(ConventionalBlockTags.PURPLE_DYED_BLOCKS, "Purple Dyed Blocks");
+		translationBuilder.add(ConventionalBlockTags.RED_DYED_BLOCKS, "Red Dyed Blocks");
+		translationBuilder.add(ConventionalBlockTags.WHITE_DYED_BLOCKS, "White Dyed Blocks");
+		translationBuilder.add(ConventionalBlockTags.YELLOW_DYED_BLOCKS, "Yellow Dyed Blocks");
+		translationBuilder.add(ConventionalBlockTags.RELOCATION_NOT_SUPPORTED, "Relocation Not Supported");
+
+		// Block Entity Types
+		translationBuilder.add(ConventionalBlockEntityTypeTags.RELOCATION_NOT_SUPPORTED, "Relocation Not Supported");
+
+		// Items
+		translationBuilder.add(ConventionalItemTags.TOOLS, "Tools");
+		translationBuilder.add(ConventionalItemTags.SHEARS_TOOLS, "Shears");
+		translationBuilder.add(ConventionalItemTags.SPEARS_TOOLS, "Spears");
+		translationBuilder.add(ConventionalItemTags.BOWS_TOOLS, "Bows");
+		translationBuilder.add(ConventionalItemTags.CROSSBOWS_TOOLS, "Crossbows");
+		translationBuilder.add(ConventionalItemTags.SHIELDS_TOOLS, "Shields");
+		translationBuilder.add(ConventionalItemTags.FISHING_RODS_TOOLS, "Fishing Rods");
+		translationBuilder.add(ConventionalItemTags.DUSTS, "Dusts");
+		translationBuilder.add(ConventionalItemTags.GEMS, "Gems");
+		translationBuilder.add(ConventionalItemTags.INGOTS, "Ingots");
+		translationBuilder.add(ConventionalItemTags.NUGGETS, "Nuggets");
+		translationBuilder.add(ConventionalItemTags.ORES, "Ores");
+		translationBuilder.add(ConventionalItemTags.RAW_MATERIALS, "Raw Materials");
+		translationBuilder.add(ConventionalItemTags.RAW_BLOCKS, "Raw Blocks");
+		translationBuilder.add(ConventionalItemTags.IRON_RAW_MATERIALS, "Raw Materials Iron");
+		translationBuilder.add(ConventionalItemTags.GOLD_RAW_MATERIALS, "Raw Materials Gold");
+		translationBuilder.add(ConventionalItemTags.COPPER_RAW_MATERIALS, "Raw Materials Copper");
+		translationBuilder.add(ConventionalItemTags.IRON_RAW_BLOCKS, "Raw Blocks Iron");
+		translationBuilder.add(ConventionalItemTags.GOLD_RAW_BLOCKS, "Raw Blocks Gold");
+		translationBuilder.add(ConventionalItemTags.COPPER_RAW_BLOCKS, "Raw Blocks Copper");
+		translationBuilder.add(ConventionalItemTags.IRON_INGOTS, "Ingots Iron");
+		translationBuilder.add(ConventionalItemTags.GOLD_INGOTS, "Ingots Gold");
+		translationBuilder.add(ConventionalItemTags.COPPER_INGOTS, "Ingots Copper");
+		translationBuilder.add(ConventionalItemTags.NETHERITE_INGOTS, "Ingots Netherite");
```

Comment:
These translations are reversed.

```suggestion
		translationBuilder.add(ConventionalItemTags.IRON_RAW_MATERIALS, "Raw Iron Materials");
		translationBuilder.add(ConventionalItemTags.GOLD_RAW_MATERIALS, "Raw Gold Materials");
		translationBuilder.add(ConventionalItemTags.COPPER_RAW_MATERIALS, "Raw Copper Materials");
		translationBuilder.add(ConventionalItemTags.IRON_RAW_BLOCKS, "Raw Iron Blocks");
		translationBuilder.add(ConventionalItemTags.GOLD_RAW_BLOCKS, "Raw Gold Blocks");
		translationBuilder.add(ConventionalItemTags.COPPER_RAW_BLOCKS, "Raw Copper Blocks");
		translationBuilder.add(ConventionalItemTags.IRON_INGOTS, "Iron Ingots");
		translationBuilder.add(ConventionalItemTags.GOLD_INGOTS, "Gold Ingots");
		translationBuilder.add(ConventionalItemTags.COPPER_INGOTS, "Copper Ingots");
		translationBuilder.add(ConventionalItemTags.NETHERITE_INGOTS, "Netherite Ingots");
```

Reply:
Also should be resolved too. Hopefully rest of translations are good if I haven't gone cross-eyed.